### PR TITLE
[WFLY-15441] Provide Native Jakarta namespace WildFly Elytron artifacts

### PR DESCRIPTION
**Pull request by ropalka**
https://issues.redhat.com/browse/WFLY-15441

### CODE REVIEW

**Comment by ropalka** [comment]
@fjuma please review

File: jakarta/auth/jaspi/pom.xml [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.wildfly.security</groupId>
+        <artifactId>wildfly-elytron-parent-jakarta</artifactId>
+        <version>1.18.1.CR1-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>wildfly-elytron-jaspi-jakarta</artifactId>
```
Comment (darranl) [inline jakarta/auth/jaspi/pom.xml:33]: We don't want transformed versions of this component as we already have native versions.

File: jakarta/jacc/pom.xml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+    <parent>
+        <groupId>org.wildfly.security</groupId>
+        <artifactId>wildfly-elytron-parent-jakarta</artifactId>
+        <version>1.18.1.CR1-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>wildfly-elytron-jacc-jakarta</artifactId>
+    <name>WildFly Elytron - JACC (Jakarta Namespace)</name>
```
Comment (darranl) [inline jakarta/jacc/pom.xml:34]: This is another one we would not need a transformed version of as we have a native implementation.

**Comment by ropalka** [comment]
Closing - superseded by https://github.com/wildfly-security/wildfly-elytron/pull/1633

**Pull request closed without merging**